cosmos/lib/cosmos/api/tlm_api.rb

```diff
@@ -66,90 +55,30 @@ module Api
-    # @return [Numeric] The unconverted telemetry value without formatting or
-    #   units
     def tlm_raw(*args, scope: $cosmos_scope, token: $cosmos_token)
-      target_name, packet_name, item_name = tlm_process_args(args, 'tlm_raw', scope: scope)
-      authorize(permission: 'tlm', target_name: target_name, packet_name: packet_name, scope: scope, token: token)
-      CvtModel.get_item(target_name, packet_name, item_name, type: :RAW, scope: scope)
+      tlm(*args, type: :RAW)
     end
-
     # Request a formatted telemetry item from a packet.
-    #
-    # Accepts two different calling styles:
-    #   tlm_formatted("TGT PKT ITEM")
-    #   tlm_formatted('TGT','PKT','ITEM')
-    #
-    # Favor the first syntax where possible as it is more succinct.
-    #
-    # @param args (see #tlm)
-    # @return [String] The converted telemetry value with formatting but
-    #   without units
     def tlm_formatted(*args, scope: $cosmos_scope, token: $cosmos_token)
-      target_name, packet_name, item_name = tlm_process_args(args, 'tlm_formatted', scope: scope)
-      authorize(permission: 'tlm', target_name: target_name, packet_name: packet_name, scope: scope, token: token)
-      CvtModel.get_item(target_name, packet_name, item_name, type: :FORMATTED, scope: scope)
+      tlm(*args, type: :FORMATTED)
     end
-
     # Request a telemetry item with units from a packet.
-    #
-    # Accepts two different calling styles:
-    #   tlm_with_units("TGT PKT ITEM")
-    #   tlm_with_units('TGT','PKT','ITEM')
-    #
-    # Favor the first syntax where possible as it is more succinct.
-    #
-    # @param args (see #tlm)
-    # @return [String] The converted, formatted telemetry value with units
     def tlm_with_units(*args, scope: $cosmos_scope, token: $cosmos_token)
-      target_name, packet_name, item_name = tlm_process_args(args, 'tlm_with_units', scope: scope)
-      authorize(permission: 'tlm', target_name: target_name, packet_name: packet_name, scope: scope, token: token)
-      CvtModel.get_item(target_name, packet_name, item_name, type: :WITH_UNITS, scope: scope)
+      tlm(*args, type: :WITH_UNITS)
     end
-
-    # Request a telemetry item from a packet with the specified conversion
-    # applied. This method is equivalent to calling the other tlm_xxx methods.
-    #
-    # Accepts two different calling styles:
-    #   tlm_variable("TGT PKT ITEM", :RAW)
-    #   tlm_variable('TGT','PKT','ITEM', :RAW)
-    #
-    # Favor the first syntax where possible as it is more succinct.
-    #
-    # @param args The args must either be a string followed by a symbol or
-    #   three strings followed by a symbol (see the calling style in the
-    #   description). The symbol must be one of {Packet::VALUE_TYPES}.
-    # @return [Object] The converted telemetry value
+    # Request a telemetry item with units from a packet.
+    # @deprecated Use tlm() with type keyword
     def tlm_variable(*args, scope: $cosmos_scope, token: $cosmos_token)
```

Comment:
This could probably be removed. I'm not sure how many people use it. But it was easy to keep as a deprecated method for now.